erpnext_com/www/docs/user/manual/en/support/issue.md

```diff
@@ -16,13 +16,25 @@ Frappe system.
 
 <img class="screenshot" alt="Issue" src="{{docs_base_url}}/assets/img/support/issue.png">
 
```

Comment:
the GIFs have extra width, crop

Reply:
clubed both gifs into one, for better understanding